### DISCUSSION TITLE
DEVNET - Token Unknow and couldn't fetch network fee

### DISCUSSION THREAD

**Discussion by BlumpSol** [phantom/docs#264] [Help & Support] [answered]
### Summary

I can't get the transaction fees or the token name when testing on DEVNET (I've been testing for several days).

### Steps to Reproduce

Hi, 
I can't get the transaction fees or the token name when testing on DEVNET (I've been testing for several days). However, the transaction is correct and sent when I click on confirm. 
I tested with a spl token created on the DEVNET network but also with USDC DEVNET. Always the same result. 
Can anyone tell me more about this? 

Thx ! 
<img width="357" alt="Capture d’écran 2024-07-30 à 02 39 26" src="https://github.com/user-attachments/assets/176373f7-b506-4c7c-ad23-44b0c6c309b8">

### Phantom Version

24.11.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hello @BlumpSol,

We are investigating an issue with simulation and fee estimation on devnet.

**Comment by elemanhillary** [comment]
<img width="1728" alt="Screenshot 2024-08-12 at 6 50 28 PM" src="https://github.com/user-attachments/assets/905e43ba-0f37-4eef-b0be-4b155a3b8386">
Same issue here.

**Comment by wmemon** [comment]
Hi, did anyone find the error? I have been facing the same issue..

**Comment by Slitthe** [comment]
Any updates on this? I'm also getting this error

**Comment by elemanhillary** [comment]
Mine worked with custom RPC url

**Comment by adamdelphantom** [comment]
Fix on this issue should be available in the coming weeks. Thanks for everyone's patience!

@BlumpSol @elemanhillary @Slitthe @wmemon

**Comment by Harushii18** [comment]
This occurs on testnet as well, just to add to it @adamdelphantom

**Comment by salmandayal** [comment]
Facing the same issue. Are you facing the same issue on the mainnet as well with your custom contract?  @Harushii18  @Slitthe

**Reply by adamdelphantom** [reply]
Hey @salmandayal,

This specific issue is isolated to devnet. If you're seeing this on main net please start a new thread with all relevant info, so we can try to help.

**Reply by Harushii18** [reply]
Haven't been able to test with mainnet, only testnet and devnet. This happens for both networks on the phantom desktop extension for me. I'm using solana wallet adapter to trigger the extension to pop up, and the payment goes through however it does as the earlier screenshot posted shows: does not want to show the network fee and gives that warning that the transaction may fail.

**Reply by geoshak** [reply]
Similar issue here.

**Comment by adamdelphantom** [comment, marked as answer]
@BlumpSol @elemanhillary @wmemon @Slitthe @Harushii18 @geoshak Thanks for your patience. This should be resolved on version 24.19. Please let me know if not.

**Reply by Harushii18** [reply]
Thank you for your prompt responses!

**Comment by tony-yyj** [comment]
still facing the same issue
Phantom version: 24.19
![2024-09-30 17 38 57](https://github.com/user-attachments/assets/fabe436b-c4bc-419b-8faf-437bb9d19bdc)

**Reply by adamdelphantom** [reply]
@tony-yyj What network is this on, please?

**Reply by tony-yyj** [reply]
devnet

**Reply by adamdelphantom** [reply]
@tony-yyj, Would you mind providing some of your code, please? Our [sandboxes example app](https://phantom-sandboxes.vercel.app/) which was previously exhibiting the same error is now working.

**Comment by charlie83xt** [comment]
Was this ever resolved?
